src/test/java/com/wikia/webdriver/common/dataprovider/ads/FandomAdsDataProvider.java

```diff
@@ -50,4 +52,41 @@ private FandomAdsDataProvider() {
         }
     };
   }
+
+    @DataProvider
+    public static Object[][] fandomVuapPage() {
+        return new Object[][]{
+                {
+                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
+                        FandomAdsDataProvider.VUAP_PAGE_SLUG,
+                        AdsFandomContent.GPT_TOP_LEADERBOARD,
+                        String.format(AD_UNIT_TEMPLATE, AdsFandomContent.TOP_LEADERBOARD)
+                },
+                {
+                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
+                        VUAP_PAGE_SLUG,
+                        AdsFandomContent.GPT_BOTTOM_LEADERBOARD,
+                        String.format(AD_UNIT_TEMPLATE, AdsFandomContent.BOTTOM_LEADERBOARD)
+                }
+        };
+    }
+
+    @DataProvider
+    public static Object[][] fandomVuapPageMobile() {
+        return new Object[][]{
+                {
+                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
+                        FandomAdsDataProvider.VUAP_PAGE_SLUG,
+                        AdsFandomContent.GPT_TOP_LEADERBOARD,
+                        String.format(AD_UNIT_TEMPLATE, AdsFandomContent.TOP_LEADERBOARD)
+                }
+//  Currently on Fandom mobile BFFA is not displayed ADEN-4479 waiting for fix
+//                {
+//                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
+//                        FandomAdsDataProvider.VUAP_PAGE_SLUG,
+//                        AdsFandomContent.GPT_BOTTOM_LEADERBOARD_MOBILE,
+//                        String.format(AD_UNIT_TEMPLATE, AdsFandomContent.BOTTOM_LEADERBOARD)
+//                }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)